lib/ui/compositing.dart

```diff
@@ -197,6 +197,12 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   }
   void _addPicture(double dx, double dy, Picture picture, int hints) native "SceneBuilder_addPicture";
 
+  void addTexture(int textureId, {Offset offset: Offset.zero, double width: 0.0, double height: 0.0}) {
+   _addTexture(offset.dx, offset.dy, width, height, textureId);
```

Comment:
assert that offset isn't null (i think we have a helper function to do that) before you dereference it